src/lib/helpers/formatCurrency.ts

```diff
@@ -3,20 +3,26 @@ import { formatUnits } from "ethers/lib/utils.js"
 import { localeNumber } from "@/lib/helpers"
 
 export const formatCurrencyUnits = ({
-  abbreviate = false,
   amountWei = "0",
   decimals = 18,
+  maximumFractionDigits,
 }: {
-  abbreviate?: boolean
   amountWei?: string
   decimals?: number
-}) =>
-  abbreviate
-    ? localeNumber(Number(formatUnits(amountWei, decimals)), {
-        compact: true,
-        maximumFractionDigits: 2,
-      })
-    : formatUnits(amountWei, decimals)
+  maximumFractionDigits?: number
+}) => {
+  if (maximumFractionDigits !== undefined) {
+    const formatted = formatUnits(amountWei, decimals)
+    return formatted.startsWith("-")
```

Comment:
I don't think there is any case in which a currency value should be negative. This simplifies a bunch of precision-loss related negative numbers when calculating changes from lever actions.